react-design/my-app/src/RotatingBanner.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import './RotatingBanner.css';
+
+export default function RotatingBanner({ items }) {
+  const [index, setIndex] = useState(0);
+
+  function handleNext() {
+    setIndex((index + 1) % items.length);
+  }
+
+  function handlePrev() {
+    setIndex((index - 1 + items.length) % items.length);
+  }
+
+  function handleIndicate(currentIndex) {
+    console.log(2);
+    setIndex(currentIndex);
+  }
+
+  return (
+    <div>
+      <Banner item={items[index]} />
+      <PrevButton onPrev={handlePrev} />
+      <Indicators
+        onIndicate={handleIndicate}
+        currentIndex={index}
+        count={items.length}
+      />
+      <NextButton onNext={handleNext} />
+    </div>
+  );
+}
+
+function Banner({ item }) {
+  return <h2>{item}</h2>;
+}
+
+function NextButton({ onNext }) {
+  return <button onClick={onNext}>Next</button>;
+}
+
+function PrevButton({ onPrev }) {
+  return <button onClick={onPrev}>Prev</button>;
+}
```

Comment:
Don't forget to specify the `type`s of your buttons so they don't default to `"submit"`.